WebInstall/explainer_cross_domain.md

```diff
@@ -113,7 +113,7 @@ To install a web app, a web site would use the promise-based method `navigator.i
 
 * Resolve when an installation was completed.
     * The success value will be an object that contains:
-     	*  `mode`: string with the surface-hint where the app was installed.
+     	*  `manifest_id`: string with the computed `manifest_id` of the installed app.
```

Comment:
Should you make the same change to same domain as well?

https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/743b3f6b8946c4db11be39cbf08fc4a89ce92bfc/WebInstall/explainer_same_domain.md?plain=1#L74-L76

Reply:
Ah, you do this in #737